sale_coupon_limit/models/sale_coupon_rule.py

```diff
@@ -115,11 +115,14 @@ def _compute_rule_times_used(self):
                         ]
                     )
                     continue
-                salesman_limit.rule_times_used = self.env["sale.order"].search_count(
-                    [
-                        "|",
-                        ("no_code_promo_program_ids", "in", [program["id"]]),
-                        ("code_promo_program_id", "=", [program["id"]]),
-                        ("user_id", "=", salesman_limit.rule_user_id.id),
-                    ]
+                salesman_limit.rule_times_used = len(
+                    self.env["sale.order.line"].read_group(
+                        [
+                            ("coupon_program_id", "=", [program["id"]]),
```

Comment:
Why between brackets?

Reply:
A remanent. Changed. Funny that the domain was interpreted anyway